normandy/recipes/tests/api/v3/test_serializers.py

```diff
@@ -181,3 +183,55 @@ def test_it_uses_cdn_url(self, rf, settings):
         action = ActionFactory()
         serializer = ActionSerializer(action, context={"request": rf.get("/")})
         assert serializer.data["implementation_url"].startswith(settings.CDN_URL)
+
+
+@pytest.mark.django_db()
+class TestEnabledStateSerializer:
+    def test_it_works(self):
```

Comment:
This test is a little gross, since it goes through the whole process of creating and enabling two revisions of a recipe. It was easier than trying to create `EnabledState` objects out of whole cloth though.

Reply:
i think (unfortunately) this is the best way to test thees.